apiron/client.py

```diff
@@ -207,32 +207,17 @@ def call(
     """
     logger = logger or LOGGER
 
-    if hasattr(endpoint, "stub_response"):
```

Comment:
This responsibility now falls upon the `StupEndpoint` class, giving a bit better cohesion.